src/data.js

```diff
@@ -21,6 +21,26 @@ const pages = [
       'CSS',
     ],
   },
+  {
+    name: 'Danilo Barion',
+    description:
+      'Father, developer, blog writer, classical guitar player and searching for the meaning of life!',
+    url: 'https://danilobarion.com/uses',
```

Comment:
This URL doesn't seem to work

Reply:
Sorry I've not seen your comment. I've fixed the URL already, will reopen the PR.